web-common/src/components/menu/compositions/SelectMenu.svelte

```diff
@@ -7,22 +7,15 @@ A slot is provided to change the text within the button.
 
 -->
 <script lang="ts">
-  import { createEventDispatcher, setContext } from "svelte";
+  import { createEventDispatcher } from "svelte";
   import type { Alignment, Location } from "../../floating-element/types";
   import { SelectButton, WithSelectMenu } from "../index";
+  import type { SelectMenuItem } from "../types";
 
-  export let options;
-  export let selection;
-  export let tailwindClasses = undefined;
-  export let activeTailwindClasses = undefined;
-  /** When true, will make the trigger element a block-level element.
-   * This is most useful when embedding a select menu in a table or wherever
-   * a block-level treatment is needed.
-   */
-  export let block = false;
```

Comment:
Any reason why we are removing these props?

Reply:
They are not used anywhere, so simplifying the component -- it's needless over-engineering and confusing to consumers of an API to have a bunch of extraneous options (most of which are undocumented and untyped).